src/OneBlinkFormBase.tsx

```diff
@@ -914,6 +924,45 @@ function OneBlinkFormBase({
                 You have unsaved changes, are you sure you want discard them?
               </p>
             </Modal>
+            <Modal
+              isOpen={promptUploadingAttachments === true}
+              title="Attachment upload in progress"
+              cardClassName="cypress-attachments-wait-continue"
+              titleClassName="cypress-attachments-confirm-wait-title"
+              bodyClassName="cypress-attachments-confirm-wait-body"
+              actions={
+                <>
+                  <span style={{ flex: 1 }}></span>
+                  <button
+                    type="button"
+                    className="button ob-button is-light cypress-attachments-confirm-wait"
+                    onClick={handleWaitForAttachments}
+                  >
+                    <CustomisableButtonInner
+                      label={'Wait'}
+                      icon={buttons?.cancelPromptNo?.icon}
```

Comment:
If we are going to add customisable icons here, we prob want to add them to the console, not sure these icons will suit this purpose as well.